server/routes/superheroes_routes.js

```diff
@@ -0,0 +1,91 @@
+const superheroesRouter = require('express').Router();
+const { Superhero } = require('../db/models/models_index.js');
+
+// app.use('/api/superheroes', superheroesRouter) in routes_index.js
+
+// get all superheroes
+superheroesRouter.get('/', async (req, res) => {
+    
+    try {
+        const superheroes = await Superhero.findAll();
+        res.status(200).send({ superheroes });
+    } catch(e) {
+        console.log(e);
+        res.status(400).send({ message: 'Superheroes not found' });
+    }
+});
+
+// get individual superhero
+superheroesRouter.get('/:id', async (req, res) => {
+    const { id } = req.params;
+
+    try {
+        const superhero = await Superhero.findByPk(id);
+        res.status(200).send({ superhero });
+    } catch (e) {
+        console.log(e);
+        res.status(400).send({ message: `Superhero id: ${id} not found.` })
+    }
+});
+
+// update/edit superhero
+superheroesRouter.put('/:id', async (req, res) => {
+    const { id } = req.params;
+    const { availability, categoryId, description, image, name, powers, offerings, price } = req.body; // review to agree on properties for superhero
+
+    try {
+        const superhero = await Superhero.findByPk(id);
+        const updatedSuperhero = await superhero.update({
```

Comment:
i think you should check first if you have superhero. So just wrap const updatedSuperhero = await superhero.update({...}) into if (superhero) { ... }